packages/eslint-plugin-pf-codemods/src/ruleCustomization.ts

```diff
@@ -77,4 +77,9 @@ export const cleanupRules = [
   "no-unused-imports-v5",
   "no-unused-imports-v6",
   "no-duplicate-import-specifiers",
+
+  // These rules rename components. These components also have mods renaming props on them.
+  // -> we have to rename the component after the rules renaming props finish
```

Comment:
Rather than this, could we update the rename props rule(s) for these components to target either the old or new name?